src/ui/components/ShareQR/MoreOptions.tsx

```diff
@@ -11,34 +11,21 @@ const MoreOptions = ({
   onClick?: () => void;
 }) => {
   return (
-    <IonGrid>
-      <IonRow>
-        <IonCol size="12">
-          <span
-            className="share-qr-modal-option"
-            data-testid="share-qr-modal-share-button"
-            onClick={async () => {
-              if (onClick) onClick();
-              await Share.share({
-                text: text,
-              });
-            }}
-          >
-            <span>
-              <IonButton shape="round">
-                <IonIcon
-                  slot="icon-only"
-                  icon={openOutline}
-                />
-              </IonButton>
-            </span>
-            <span className="share-qr-modal-info-block-data">
-              {i18n.t("shareqr.more")}
-            </span>
-          </span>
-        </IonCol>
-      </IonRow>
-    </IonGrid>
+    <OptionList
+      data={[
+        {
+          testId: "share-qr-modal-share-button",
+          onClick: async () => {
```

Comment:
I know this was here before, but can we extract this into its own const e.i. `handleClick` 🙏